be/src/glibc-compatibility/glibc-compatibility.c

```diff
@@ -175,6 +175,15 @@ void __explicit_bzero_chk(void * buf, size_t len, size_t unused)
     explicit_bzero(buf, len);
 }
 
+int snprintf(char* __restrict __s, size_t __maxlen, const char* __restrict __format, ...);
```

Comment:
where does this symbol link to? add a comment? maybe explicit add a `extern` modifier is a good idea